File: Goodstein/src/com/priorsoftware/goodstein/TestGoodstein.java
package com.priorsoftware.goodstein;

public class TestGoodstein {

	public static void main(final String[] args) {
		final int n = Integer.valueOf(args[0]);
		final Goodstein g = new Goodstein(n);
		
		for (int i = 0; i < 100; i++) {
			System.out.println(g);
			g.next();
		}
	}

}

File: GoodsteinScala/src/com/priorsoftware/goodstein/scala/TestGoodstein.scala
package com.priorsoftware.goodstein.scala

object TestGoodstein {

  def main(args: Array[String]): Unit = {
    val n = args(0).toInt
    val g = new Goodstein(n);

    for (i <- 0 until 100) {
      println(g);
      g.next();
    }
  }
}
